src/content/snapshot/branching-and-baselines.md

```diff
@@ -261,10 +328,29 @@ Chromatic will now compare the stories from `q` to the corresponding stories in
 
 If the head branch has been more recently updated from the base branch, the merge base can be a more recent commit than the point we branched off:
 
-```shell
+<!-- ```shell
```

Comment:
So that we don't forget, do you plan on leaving the original graphs, or are we going to remove them? I have no issues with leaving them so that contributors have a sense of the graph itself

Reply:
Yea, I'll delete them. I wanted to leave them in for review.